src/main/java/com/cebanova/classesandobjects/autoservicetask/ManageAutoService.java

```diff
@@ -0,0 +1,75 @@
+                + AcoolaService.carCapacity);
+
+        Garage HelpAuto = new Garage("Ion Creanga 20");
+        HelpAuto.numberId = "1";
+        HelpAuto.workSchedule = "Non stop";
+        HelpAuto.hasComputerDiagnostics = true;
+        HelpAuto.carCapacity = 10;
+
+        System.out.println("HelpAuto are urmatoarele proprietati: Adresa: " + HelpAuto.address + "; Numarul: "
+                + HelpAuto.numberId + "; Programul de lucru: " + HelpAuto.workSchedule
+                + "; Diagnosticare Computerizata: " + HelpAuto.hasComputerDiagnostics + "; Capacitatea: "
+                + HelpAuto.carCapacity);
+
+        Car Ferrari = new Car("Ferrari", "488 GTB", "black");
+        Ferrari.kmWhenEnteredTheService = 50000;
+        Ferrari.productionYear = 2022;
+        Ferrari.fuelType = "gasoline";
+        Ferrari.gearboxType = "automatic";
+        Ferrari.carPrice = 459690;
+        Ferrari.plateNumber = "FFF 000";
+
+        System.out.println("A fost creata o masina noua cu urmatoarele proprietati: Marca: " + Ferrari.make + "; Modelul: "
+                + Ferrari.model + "; Culoarea: " + Ferrari.color + "; KM la bord: " + Ferrari.kmWhenEnteredTheService
+                + "; Anul producerii: " + Ferrari.productionYear + "; Tipul combustibilului: " + Ferrari.fuelType
+                + "; Tipul cutiei de viteze: " + Ferrari.gearboxType + "; Pretul masinii: " + Ferrari.carPrice
+                + "; Numarul de inmatriculare " + Ferrari.plateNumber);
+
+        Car Suzuki = new Car("SSS 000");
+        Suzuki.make = "Suzuki";
+        Suzuki.model = "Jimny";
+        Suzuki.color = "yellow";
+        Suzuki.kmWhenEnteredTheService = 90000;
+        Suzuki.productionYear = 2021;
+        Suzuki.fuelType = "gasoline";
+        Suzuki.gearboxType = "automatic";
+        Suzuki.carPrice = 30490;
+
+        System.out.println("A fost creata o masina noua cu urmatoarele proprietati: Marca: " + Suzuki.make + "; Modelul: "
+                + Suzuki.model + "; Culoarea: " + Suzuki.color + "; KM la bord: " + Suzuki.kmWhenEnteredTheService
+                + "; Anul producerii: " + Suzuki.productionYear + "; Tipul combustibilului: " + Suzuki.fuelType
+                + "; Tipul cutiei de viteze: " + Suzuki.gearboxType + "; Pretul masinii: " + Suzuki.carPrice
+                + "; Numarul de inmatriculare " + Suzuki.plateNumber);
+
+        Tool Compressor = new Tool("Compressor");
+        Compressor.weight = 1.5;
+        Compressor.uniqueCode = 121212121;
+        Compressor.needsElectricity = true;
+
+        System.out.println("A fost creata o unealta cu urmatoarele particularitati: Nume: " + Compressor.toolName
+                + "; Codul de inventariere: " + Compressor.uniqueCode + "; Electric:" + Compressor.needsElectricity);
+
+        // Constructor fara parametri:
+        Tool Elevator = new Tool();
+        System.out.println("A fost creata o unealta fara parametri");
```

Comment:
Linia 70 din clasa ManageAutoService ar trebui mutata in clasa Tool, in interiorul constructorului fara parametri

Reply:
Done!

Mersi mult.